src/Orchard.Web/Core/Common/Views/Body-Large.Editor.cshtml

```diff
@@ -1,7 +1,14 @@
 ﻿@{
-    var htmlAttributes = new Dictionary<string, object> {
-        {"class", "text large"}
-    };
+    var htmlAttributes = (Dictionary<string, object>)Model.HtmlAttributes;
+
+    if (htmlAttributes == null) {
```

Comment:
Can you try to set the attributes even if Model.HtmlAttributes is null, meaning don't check for it, just assign it and see if that works (some magic we made in Orchard1).

Reply:
I tried, it doesn't work. 
Anyway I found a bug introduced in the last commit and I fixed it and tested everything from scratch.

Reply:
I think it's because the existing property on the shapes is named `Attributes`. Can you try to use this one and even doing something like `Model.Attributes["required"] = "required"` for instance?
And it also already has a `IList<string> Classes` property which isn't null.